big/int.go

```diff
@@ -0,0 +1,350 @@
+package big
```

Comment:
Entirely copied from go-state-types. We can discuss depending on that library later, fixing cbor-gen, etc.